sbi/inference/snle/snle_base.py

```diff
@@ -366,4 +367,10 @@ def _loss(self, theta: Tensor, x: Tensor) -> Tensor:
         Returns:
             Negative log prob.
         """
+        theta = reshape_to_batch_event(
```

Comment:
line lengths seem odd here: why is this line indented and the import above in line 20 is not?